protected/humhub/modules/comment/Module.php

```diff
@@ -97,4 +97,29 @@ public function canComment($object)
         return true;
     }
 
+    /**
+     * Checks if given content object is a subcomment
+     *
+     * @param $object
+     * @return bool
```

Comment:
I'd rather move this as static function to the comment class. Also always add `@since` (in this case 1.8) to new functions. You can also refactor it as follows:

```
public function isSubComment($object)
{
    return $object instanceof Comment && $object->object_model === Comment::class
}
```

Reply:
Done! Thank you